openshift/e2e-tests-openshift.sh

```diff
@@ -212,38 +221,32 @@ function create_knative_namespace(){
 	EOF
 }
 
-function deploy_knative_operator(){
-  local COMPONENT="knative-$1"
-  local KIND=$2
+function deploy_serverless_operator(){
+  local NAME="serverless-operator"
+  local NAMESPACE="knative-serving"
```

Comment:
We have already `SERVING_NAMESPACE` and we can use it instead of creating same value.